docs/styles/_base.scss

```diff
@@ -94,15 +96,3 @@ article {
         transparent
     );
 }
-
-footer {
```

Comment:
Extracted these spaghetti styles to `chunks/_site-footer.scss`. Ensures we're styling sematic class names instead of all footer elements 😁